<launch>
  <master auto="start"/>
  <param name="/use_sim_time" value="true" />
  <arg name="map_file" default="/home/alpsark/ros-ws/src/nav_stack_tuning/launch/map.yaml"/>
  <!-- Map server -->
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" >		
    <param name="frame_id" value ="/map"/>
	</node>

  <remap from="map" to="/map"/>
  <node name="rviz" pkg="rviz" type="rviz" output="screen" />	
  <!--My packages -->

    <param name = "/amcl/initial_pose_x" value="0.0" />
    <param name = "/amcl/initial_pose_y" value="2.0" />
    <param name = "/amcl/base_frame_id" value="OmniPlatform001_base" />
    <include file = "/home/alpsark/ros-ws/src/my_subsumption/launch/robot.launch" >
	    <arg name = "robot_name" value="OmniPlatform001"/>
	    <arg name = "scan_name" value="/OmniPlatform001/sensors/scan"/>
    </include>

    <node name="go_near_beacon" pkg="go_near_beacon" type="go_near_beacon" output="screen"/>
    <node name="order" pkg="order" type="order" output="screen"/>
    <node name="deliver_order" pkg="deliver_order" type="deliver_order" output="screen"/>

    <node name="my_subsumption" pkg="my_subsumption" type="my_subsumption" output="screen" />

    <node name="pick_up_order" pkg="pick_up_order" type="pick_up_order" output="screen" />



</launch>
